static/app/utils/useTeams.tsx

```diff
@@ -172,8 +244,28 @@ function useTeams({limit}: Options = {}) {
     }
   }
 
+  useEffect(() => {
+    // Load specified team slugs
+    if (slugs) {
+      loadTeamsBySlug();
```

Comment:
ended up changing the logic here and made it so that we always call `loadTeamsBySlug` but the function will decide what slugs to fetch based on what is already in the store.

Reply:
nice